migration_to_v2_0_tests/tests/migrated_data_test.py

```diff
@@ -96,6 +101,24 @@ def test_layer_thumbnails(client, layer):
     assert diff_pixels <= DEFAULT_THUMBNAIL_PIXEL_DIFF_LIMIT, f"diff_pixels={diff_pixels}\nimg_path={img_path}\nexp_img_path={exp_img_path}"
 
 
+@pytest.mark.usefixtures("import_publication_uuids_fixture")
+@pytest.mark.parametrize("map", MAPS_TO_MIGRATE, ids=ids_fn)
+def test_maps_internal_layers(map):
```

Comment:
I suggest to test also `used_in_maps` key of GET Workspace Layer and internal `_map_layers` key of map